scripts/is-it-pretty.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+node_modules/.bin/prettier-standard '**/*.js'
+if [ -z "$(git status --porcelain)" ]; then
+  # Working directory clean
+  exit 0
+else
+  # Uncommitted changes
+  echo 'There are javascript files in this project that do not follow the prettify standard!'
```

Comment:
Maybe we can list the files? If there are unstaged changes `git diff --name-only` would list the files with changes.